source/MaterialXGenOsl/OslSyntax.cpp

```diff
@@ -210,12 +211,12 @@ namespace
             for (size_t i = 0; i<values.size(); i++)
             {
                 ss << values[i] << ", ";
-                if (i % 3 == 0)
+                if ((i != 0) && (i % 3 == 0))
```

Comment:
Added the first 0 too soon, and missed adding the last 0. Syntax was correct but values were in the wrong place.